go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -786,8 +794,10 @@ func (throttler *Throttler) collectMySQLMetrics(ctx context.Context, tmClient tm
 
 					var throttleMetricFunc func() *mysql.MySQLThrottleMetric
 					if clusterName == selfStoreName {
+						// Throttler is probing its own tablet's metrics:
 						throttleMetricFunc = throttler.generateSelfMySQLThrottleMetricFunc(ctx, probe)
 					} else {
+						// Throttler probing other tablets:
```

Comment:
I thought that we had moved this to gRPC?

Reply:
For backwards compatibility, we need to consider the option of a `v18` `PRIMARY` with `v17` replicas: those will not be able to serve gRPC, and therefore the `v18` `PRIMARY` falls back to probing them via HTTP.